totalRP3/Modules/Register/Main/RegisterList.lua

```diff
@@ -1083,3 +1082,7 @@ TRP3_API.events.listenToEvent(TRP3_API.events.WORKFLOW_ON_LOADED, function()
 		});
 	end
 end);
+
+function getDateSettings()
```

Comment:
This needs to not be a global function; would recommend renaming it and all usages to `TRP3_API.util.getPreferredDateFormat` for now as that'll likely be what's wanted by Extended.